app/components/polls/PollPage.tsx

```diff
@@ -151,7 +152,8 @@ export function usePolledPollData(
     },
     initialData,
     refetchInterval: 15_000,
+    staleTime: 15_000,
   });
 
-  return { data };
+  return data;
```

Comment:
If you check for undefined in the hook rather than in component then you can destructure in one step in the component:

In the hook:
```
if (!data) {
  notFound();
}
return data;
```

In the component:
```
const { poll, statements, filteredStatements, statementOptions } = usePolledPollData(initialData);
```